README.md

```diff
@@ -25,20 +25,28 @@ export default {
   name: 'MyModule',
   plugins: [
     resolve({
+
+      // the fields to scan in a package.json to determine the entry point..
+      mainFields: ['module', 'main'], // Default: ['module', 'main']
```

Comment:
This deviates from the defaults we currently have. I'd consider adding `browser` to be a breaking change, and so would like to avoid that for this PR.

Reply:
Okay, let's keep it until next major.